spec/controllers/api/items_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe Api::ListsController do
+  describe 'POST create' do
+    let!(:user_1) { FactoryGirl.create(:user) }
+    let!(:list_1) { FactoryGirl.create(:list) }
+    before do
+      @request.env['HTTP_AUTHORIZATION'] = ActionController::HttpAuthentication::Basic.encode_credentials(user_1.username, user_1.password)
+      @new_item = build(:item)
+      post :create, item: { name: @new_item.name }
+    end
+    it 'returns http success' do
```

Comment:
I noticed that Hound was all over the double quotes. I'm not sure I
understand how to change this, though. I looked at the links you provided
but it looks like I have to create a new config file (ruby.yml) for my
current app. I created one, but I don't know if it helped.

On Mon, Jan 4, 2016 at 6:37 PM, David Roberts <notifications@github.com>
wrote:

> In spec/controllers/api/items_controller_spec.rb
> <https://github.com/ffquint/open-to-do-api/pull/3#discussion_r48807685>:
>
> > @@ -0,0 +1,25 @@
> > +require 'rails_helper'
> > +
> > +describe Api::ListsController do
> > +  describe 'POST create' do
> > +    let!(:user_1) { FactoryGirl.create(:user) }
> > +    let!(:list_1) { FactoryGirl.create(:list) }
> > +    before do
> > +      @request.env['HTTP_AUTHORIZATION'] = ActionController::HttpAuthentication::Basic.encode_credentials(user_1.username, user_1.password)
> > +      @new_item = build(:item)
> > +      post :create, item: { name: @new_item.name }
> > +    end
> > +    it 'returns http success' do
>
> Interesting.. the style guide I use and rubocop default is the exact
> opposite of this. You should decide which you prefer. You can flip it to
> enforce the opposite by following the instructions here
> <https://houndci.com/configuration#ruby> This should be
> <https://github.com/thoughtbot/hound/blob/master/config/style_guides/ruby.yml#L105>
> the line you need to change. You can compare it to the defualt rubocop
> version
> <https://github.com/bbatsov/rubocop/blob/master/config/default.yml#L731-L735>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ffquint/open-to-do-api/pull/3/files#r48807685>.
>